roles/installer/templates/settings/ldap.py.j2

```diff
@@ -1,3 +1,5 @@
+import ldap
+
```

Comment:
I do think it would be nice to nest this in the if. This requires some refactoring.

```jinja2
{% if ldap_cacert_ca_crt %}
import ldap

AUTH_LDAP_GLOBAL_OPTIONS = {
    ldap.OPT_X_TLS_REQUIRE_CERT: True,
    ldap.OPT_X_TLS_CACERTFILE: "/etc/openldap/certs/ldap-ca.crt"
}
{% else %}
AUTH_LDAP_GLOBAL_OPTIONS = {}
{% endif %}
```